do-catalog/src/store/actions/catalog.js

```diff
@@ -2,8 +2,9 @@ import 'whatwg-fetch';
 
 const baseUrl =
   'https://cmonteserin-do-st.carto-staging.com/api/v4/data/observatory/'; // "https://public.carto.com"
-const datasetsEndpoint = 'metadata/datasets';
 const entitiesEndpoint = 'metadata/entities';
```

Comment:
We will use the same `metadata/entities` for both datasets and geographies

Reply:
When will this new endpoint be ready?